### PR TITLE
add view portfolio feature 

### LINKED ISSUE #20: maintain a portfolio of stocks
## User Story
As a user, I want to maintain a portfolio of stocks

## Detailed Description
When users buy and sell stocks, they need to keep track of what they **currently** have. This will be a backend and frontend effort, and coordination will be needed. 

The backend here will also have to be coordinated with #14 (buy and sell), since these actions will trigger a change to the portfolio. This is, however, a corequisite (can be done simultaneously).

## corequisite (has to be done at least same time)
- #14 

## Subtasks (Please also link task PRs here)
- [x] Create Backend  @willyspinner 
- [ ] Create frontend @akanksha003

### LINKED ISSUE #39: [BUG] - time locality issues
## Description
The time uses the users local time in both frontend and backend. This is bad. We need to standardize UTC.

## Expected Behavior
Regardless of wherever the user is, always use NY time.

## Additional Context
Enter any other details such as dependencies, environment, examples, etc.

## Relevant screenshots
If applicable, add screenshots to help illustrate the bug.

### PR DESCRIPTION
**Pull request by willyspinner**
Closes story #20 . this one is the backend + frontend together.

Also closes #39, which is the time locality issue - by syncing to NY time.

### CODE REVIEW

File: client/src/pages/myProfile.jsx
Comment (willyspinner) [inline client/src/pages/myProfile.jsx:65]: Can we remove the console logs before merging to master? 🤣

**Comment by akanksha003** [comment]
The following tasks have been completed:
- A user can view their Portfolio items under their profile page (in tabular form)
- A user can now also view a pie chart showing their investment distribution 

This branch is ready for review.

**Review by willyspinner — COMMENTED** [review]
Just a minor change to be fixed @akanksha003 .

 For some reason I can't approve this. Apparently because I'm also the PR author. Looking into it right now

File: client/src/pages/myProfile.jsx
Comment (akanksha003) [reply client/src/pages/myProfile.jsx:65]: oh haha, sorry about that! I forgot to remove them after testing! I'll fix it right now

**Review by willyboom — APPROVED** [review]
LGTM

**Pull request merged**
